docs/helm/examples/with-elasticsearch-auth.md

```diff
@@ -61,7 +61,6 @@ Deploy the ACS stack with the appropriate values files.
 
 ```bash
 helm install acs ./helm/alfresco-content-services \
```

Comment:
here we are assuming user using git but most of other examples use the chart registry, they should be coherent.

this would make tricky to fetch the additional required values file, we should also improve that